### ISSUE TITLE
'long' import error

### ISSUE THREAD

**Issue by chullino** [grpc/grpc-node#2111]
### Problem description
The latest version 0.6.10 has error with importing 'long', and as a result I can not compile. Error log comes like this. 

```
node_modules/@grpc/proto-loader/build/src/index.d.ts:22:10 - error TS2617:  'Long' can only be imported by using 'import Long = require("long")' or by turning on the 'esModuleInterop' flag and using a default import.

22 export { Long } from 'long';
```

### Reproduction steps

I can compile with dependency below.
```
    "@grpc/proto-loader": {
      "version": "0.6.9",
      "resolved": "https://registry.npmjs.org/@grpc/proto-loader/-/proto-loader-0.6.9.tgz",
      "integrity": "sha512-UlcCS8VbsU9d3XTXGiEVFonN7hXk+oMXZtoHHG2oSA1/GcDP1q6OUgs20PzHDGizzyi8ufGSUDlk3O2NyY7leg==",
      "requires": {
        "@types/long": "^4.0.1",
        "lodash.camelcase": "^4.3.0",
        "long": "^4.0.0",
        "protobufjs": "^6.10.0",
        "yargs": "^16.2.0"
      }
    },
```

I cannot compile with dependency below.
```
    "@grpc/proto-loader": {
      "version": "0.6.10",
      "resolved": "https://registry.npmjs.org/@grpc/proto-loader/-/proto-loader-0.6.10.tgz",
      "integrity": "sha512-TM2ghLt8pchK+XAlytMJ7RQvSqAb75c8kdizk6PuscNOT8aLYUL/CWESGLvp5GiDsWb6Gzs/VEen1M0e43ggZw==",
      "optional": true,
      "requires": {
        "@types/long": "^4.0.1",
        "lodash.camelcase": "^4.3.0",
        "long": "^5.2.0",
        "protobufjs": "^6.10.0",
        "yargs": "^16.2.0"
      }
    },
```

### Environment
 - MacOS Big Sur
 - node version: 14.17.3
 - Node installation method : nvm

### Additional context
I had been building without any problem until yesterday. The error came out today.

**Comment by murgatroid99** [comment]
That's weird, because the only change in version 0.6.10 was to remove the line of code that appears in the quoted error message. Please double-check what version of the package is actually installed in that path.

**Comment by chullino** [comment]
IMHO, I don't think that's the only change because the dependency also changed.

```
"long": "^4.0.0" => "long": "^5.2.0"
```

I solved this problem by wiping out all the dependencies, installing @grpc/proto-loader@0.6.9.

```
rm -rf node_modules
rm package-lock.json
npm install @grpc/proto-loader@0.6.9
npm install
```

**Comment by denis-vogrincic** [comment]
This also breaks build where other packages have this as a dependency.

**Comment by murgatroid99** [comment]
Can someone please share the specific error that this causes? The quoted error message refers to a line that is not in version 0.6.10 of this package, and therefore it cannot possibly reflect a breakage in version 0.6.10 of the package.

**Comment by murgatroid99** [comment]
This should be fixed in 0.6.11. If you have a dependency on `long` version 4, proto-loader will use it instead of installing `long` version 5.
